src/repositories/user.repository.js

```diff
@@ -4,15 +4,21 @@ const Model = require('../database/models/users.model');
 module.exports = class UserRepository {
     async createUser(user) {
         try {
-            const hashedPassword = await bcrypt.hash(user.password, 10);
-            user.password = hashedPassword;
-            const newUser = await Model.create(user);
-            return {
-                user: newUser.username,
-                email: newUser.email
-            };
+            if(user.password === user.rePassword){
```

Comment:
playing around locally, this stumped for a while, there is no error message returned if the passwords don't match. Should throw an error with a clear message if they don't match

Reply:
This should be addressed though. I had to some serious debugging to see why my request wasn't getting through